docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,104 @@
+# Contributing
+
+Stellio is an open source project, currently in the incubator to become a FIWARE Generic Enabler. We welcome any contributions from everyone.  
+If you wish to contribute on our code, you should first read and follow guidelines mentioned in this document.
+
+## Ask a question 
+
+Please if you want to ask a question regarding Stellio and how to use it, create an issue
+ in [Stackoverflow](https://stackoverflow.com/) with the tag fiware-stellio.
```

Comment:
I remember seeing this Tag when preparing the Fiwrare requirements, do we need to prepare something for it ?

Reply:
I don't know, I saw that in the requirements too. I guess we have to subscribe to this tag to see questions related to Stellio.

Reply:
If you find a way to subscribe to a tag on SO, I'd like to know how :)

Reply:
With RSS you can subscribe to 
```https://stackoverflow.com/feeds/tag/fiware-stellio```
But first we need to create a question to generate the tag.

Reply:
I found also in Stackoverflow that it should be working with "Watched Tags", but I ask to see it working. (just subscribed so I don't know if it's going to work) from https://meta.stackoverflow.com/questions/254318/how-to-subscribe-to-tags
![image](https://user-images.githubusercontent.com/11096082/88644491-531f9400-d0c3-11ea-9044-e4f6f343551c.png)